plugin/notes.lua

```diff
@@ -0,0 +1,24 @@
+
+-- Add the plugin root to package.path to find our lua modules
+local plugin_root = vim.fn.fnamemodify(debug.getinfo(1, "S").source:sub(2), ":p:h:h")
```

Comment:
this doesn't look like the correct way to have a simple neovim plugin. Double check this please.